src/Core/src/Platform/Android/TextViewExtensions.cs

```diff
@@ -21,12 +21,17 @@ public static void UpdateTextHtml(this TextView textView, ILabel label)
 			var text = label.Text ?? string.Empty;
 			var htmlText = WebUtility.HtmlDecode(text);
 
-			if (OperatingSystem.IsAndroidVersionAtLeast(24))
-				textView.SetText(Html.FromHtml(htmlText, FromHtmlOptions.ModeCompact), BufferType.Spannable);
-			else
+			void SetTextHtml()
+			{
+				if (OperatingSystem.IsAndroidVersionAtLeast(24))
```

Comment:
I know images already works on iOS and this change align the behavior. Not sure about the Windows behavior, did you try?

Reply:
I don't have a Windows device. Maybe you could have a look at it, please?